communities/hacs/community-impact-proposal.md

```diff
@@ -0,0 +1,133 @@
+
+We would like to therefore try harder to maintain the excitement and activity within the group. Im not quite sure how exactly we can go about doing that at this time, but I would like to look into it.
+
+> I propose we plan out our events up to a month in advance in order to be able to better market and promote our events to carry forward momentum from the first week of university.
+
+### Delivering our values
+
+One of our goals should always be to provide meaningful and valuable events and workshops to our members. Our community puts our *members* first. Everything we do should be for the benefit of our current and future members.
+
+We should aim to continue providing workshops, talks and socials for our members. Helping to increase their employability as well as providing them with a fun and safe social environment to interact in.
+
+Our values and goals include
+
+- Helping our members to become more employable
+- Helping our members to build
+- Helping our members to learn
+- Helping our members to make new friends
+
+In order to maintain these goals and values I propose
+
+> We continue to organise workshops and talks for our members. Introduce our members to more hackathons and organise trips to them. Facilitate a community of building and sharing to organically build projects in. Organise socials so that our members can interact face to face.
+
+In addition to the build point I'd like to make it easier for members to contribute directly to the society so for that reason
+
+> I propose we make our website more open and people can contribute to it and improve it.
+
+This is already the case and has happened quite organically. Our site exists on GitHub and one of our members has been maintaining it without even been asked. I'd like to improve this and make it easier for everyone to contribute to by outlining some build steps and hosting it properly and adding in some automation.
+
+### Summary of Goals
+
+- Document procedures, ideas, ideology of the community to ensure long term success.
+- Reduce the attrition rate of members over the academic year.
+- Expand beyond just CS students to include more Art/Media and Networking students.
+- Continue to provide value to our members by running events, workshops and socials
+
+
+## Your Proposal's Execution Plan
+
+### Documentation
+
+In order to document all our procedures, ideas etc, I will need to go through all our old notes from meetings. From those I can compile a list of useful ideas and discussions to pass down to each new society. I should also outline this procedure so that new communities can do the same at infinitum.
+
+### Maintaining members
+
+This one I don't have a definite plan for so for that reason, we'd have to sit down with the rest of the committee and outline some plans for how to combat this. One way may be to be more organised just to carry forward the momentum from the society fair.
```

Comment:
This could be a tough meeting, I can see this immediately branching into a million sub-discussions. It might be a good idea to think about prompts for the possible approaches to this issue, outlining a clear agenda, and railroading this meeting hard. 

Has this meeting happened already?